### PR TITLE
Feature/issue 69 

### PR DESCRIPTION
**Pull request by ajay74984**
please review.
I have also added test cases for #122 and some other cases as well for failing tests.

### CODE REVIEW

**Comment by magwas** [comment]
call me if you need help with the build failure

**Comment by magwas** [comment]
You have mutation coverage holes.

File: src/test/java/com/kodekonveyor/authentication/UserEntityTestData.java [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
   public static final String BAD_LOGIN = "424241";
   public static final long USER_ID = 4242;
+  public static final long CUSTOMER_ID = 4242;
   public static final long BAD_USER_ID = 4241;
   public static final String NO_WORKREQUESTS_ID = "4243";
   public static final String OWNER_ID = "4242";
   public static final String NEGATIVE_OWNERID_ID = "-4536";
   public static final Long NO_WORKREQUESTS_ID_ASLONG =
       Long.parseLong(NO_WORKREQUESTS_ID);
+  public static final long PROVIDER_ID = 4242;
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/authentication/UserEntityTestData.java:21]: CUSTOMER_ID and PROVIDER_ID are the same values for the same thing. Please get rid of one of them.

File: src/test/java/com/kodekonveyor/work_request/find/FindTestHelper.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.stream.Collectors;
+
+import com.kodekonveyor.work_request.WorkRequestDTO;
+import com.kodekonveyor.work_request.WorkRequestEntityTestData;
+
+public class FindTestHelper {
+
+  public static void
+      assertContains(final List<WorkRequestDTO> container) {
+    assertEquals(
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/find/FindTestHelper.java:16]: I believe this assertion is not needed, the other one checks correctly. Please get rid of it, and the associated constant.

File: src/test/java/com/kodekonveyor/authentication/UserEntityTestData.java (hunk 1 above)
Comment (ajay74984) [reply src/test/java/com/kodekonveyor/authentication/UserEntityTestData.java]: @magwas incorporated it in the latest commit.

File: src/test/java/com/kodekonveyor/work_request/find/FindTestHelper.java (hunk 2 above)
Comment (ajay74984) [reply src/test/java/com/kodekonveyor/work_request/find/FindTestHelper.java]: @magwas incorporated it in the latest commit.

File: src/test/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsControllerDataAccessTest.java [hunk 3]
```diff
@@ -1,72 +1,81 @@
-package com.kodekonveyor.work_request;
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsControllerDataAccessTest.java:1]: please use unix line endings

Comment (ajay74984) [reply src/test/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsControllerDataAccessTest.java]: should be ok now

**Comment by ajay74984** [comment]
thanks. But there are 7 test cases, not 6.

**Comment by magwas** [comment]
testcases=7

**Pull request merged**